content/ux-strat-bootcamp/week-one/friday/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Friday
+ready: true
+---
+
+# **Dashboard Analysis and Visualisation** 
```

Comment:
If you look at the web page that gets generated from this file it will look like this:

# Friday 
# Dashboard Analysis and Visualisation 
[Dashboard Analysis and Visualisation](http://somewhere)

To me this looks repetitive repetitive. Are you sure you want it to say the same thing twice?